app/messaging/message-processor.js

```diff
@@ -1,9 +1,52 @@
-import * as util from 'util'
 import { validateStatusMessageRequest } from './validate-inbound-message.js'
+import { CLAIM_STATUS } from '../constants.js'
+import { getLatestContactDetails } from '../api/application-api.js'
+import { config } from '../config/index.js'
+import { getByClaimRefAndMessageType, set } from '../repositories/message-generate-repository.js'
+import { sendEvidenceEmail } from '../email/evidence-email.js'
+
+const processInCheckStatusMessage = async (message, logger) => {
+  if (!config.evidenceEmail.enabled) {
+    return
+  }
+  const { claimStatus, agreementReference, claimReference, sbi, crn } = message.body
+  const messageType = `statusChange-${claimStatus}`
+
+  const messageGenerate = await getByClaimRefAndMessageType(claimReference, messageType)
+
+  if (!messageGenerate) {
+    const contactDetails = await getLatestContactDetails(agreementReference, logger)
```

Comment:
Not sure this plugs in correctly. The response here would return something that results in either 1 or 2 sets of contact details (farmer name and email AND/OR org name and email). Would expect us to then call through to sendEvidenceEmail once or twice depending on what was in the payload coming back here. Think we should be handling that now, and then just logging out somewhere where we would be sending to if it was switched on, so we can see in logs we picked this up and recognised successfully.